settings.php

```diff
@@ -29,6 +29,9 @@
 use \MXTranslator\Controller as translator_controller;
 
 if ($hassiteconfig) {
+	// get list of user map settings for account association
+	$accountOpts = array_merge(['id'], \core_user::AUTHSYNCFIELDS);
```

Comment:
Assuming something is wrong here since attempting to use `AUTHSYNCFIELDS` throws an error.